src/components/HomePage/PaginationComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+type PaginationComponentProps = {
+}
+type PaginationComponentState = {
+  currentPage: number,
+  totalPages: number,
+  // totalItems : number,
+  pageSize: number,
+  pages: number[]
+}
+
+class PaginationComponent extends React.Component<PaginationComponentProps, PaginationComponentState> {
+  constructor(props: PaginationComponentProps) {
+    super(props);
+
+    this.state = {
+      currentPage: 2,
+      totalPages: 10,
+      pageSize: 8,
+      pages: [1, 2],
+    };
+
+    this.changePage = this.changePage.bind(this);
+  }
+
+  changePage(page: number) {
+    this.setState({
+      // todo: update logic in the next task with state
+    });
+  }
+
+  render() {
+    // const { someNumbersList, currentPage } = this.state;
+    return (
+
+      <Pagination id="nav-pagination">
+        <PaginationItem>
+          <PaginationLink className="pagination-link-style-arrow" previous />
+        </PaginationItem>
+        {/* Now the is just mock data in UI. Will be changed in the next tasks */}
+        <PaginationItem
```

Comment:
Let's move PaginationItem with PaginationLink to separate FC and use something like this:
`pages.map((item) => <Pagination item={item}>)`

Reply:
Done